apps/server/src/modules/h5p-editor/contentStorage/contentStorage.ts

```diff
@@ -37,137 +32,164 @@ export class ContentStorage implements IContentStorage {
 		if (contentId === null || contentId === undefined) {
 			contentId = await this.createContentId();
 		}
+		const contentPath = path.join(this.getContentPath(), contentId.toString(), 'content.json');
```

Comment:
node:path might not be the best option here, since this will generate different paths depending on the OS (developers on windows might get problems)